CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+-   Configurable hook script for recording synchronization ([#1484], [#1604])
+
```

Comment:
Add the `withoutOverlapping` change to the `### Changed` section and explain what it does